test/providers/summary/fossology.js

```diff
@@ -394,7 +394,8 @@ describe('FOSSologySummarizer fixtures', () => {
         }
         //{ path: 'CONTRIBUTING.md' },
         //{ path: 'gemfiles/Gemfile.test-unit.2.0.1' }
-      ]
+      ],
+      licensed: { declared: 'MIT' }
```

Comment:
existing test data has a COPYING.md file that is now considered a proper license